compiler/src/iree/compiler/Codegen/Sandbox/LinalgTensorCodegenDriver.cpp

```diff
@@ -138,6 +140,40 @@ getTileAndFuseOptionsFromConfig(func::FuncOp funcOp, int64_t tilingLevel) {
   return options;
 }
 
+/// Default method to initialize the split reduction size and unroll factor in
+/// IREE. These could be overriden by the command line options if specified.
+static bool getSplitReductionOptionsFromConfig(func::FuncOp funcOp,
+                                               int64_t &size,
+                                               uint64_t &unrollFactor) {
+  FailureOr<Operation *> rootOp = getRootOp(funcOp);
+  if (failed(rootOp)) {
+    return false;
+  }
+  auto configAttr = mlir::iree_compiler::getLoweringConfig(rootOp.value());
+  if (!configAttr) return false;
+  auto tileSizesAttr = configAttr.getTileSizes();
+  if (!tileSizesAttr || tileSizesAttr.empty()) return false;
+  // The last one is the reduction dimension.
+  auto reductionSizesAttr = tileSizesAttr[tileSizesAttr.size() - 1];
```

Comment:
Probably better to explicitly pass in the position of the tile size tuple being used. Its an implementation detail that its the last tile level that is being used, but thats not always true. I think the position in the `tileSizesAttr` used should be an input to the pass itself so that it can be controlled from the pass pipeline.